selfdrive/car/toyota/interface.py

```diff
@@ -46,7 +46,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       ret.wheelbase = 2.78
       ret.steerRatio = 17.4
       tire_stiffness_factor = 0.5533
-      ret.mass = 4387. * CV.LB_TO_KG + STD_CARGO_KG
+      ret.mass = 3340. * CV.LB_TO_KG + STD_CARGO_KG  # from Toyota eBrochures 
```

Comment:
no need for the comment
```suggestion
      ret.mass = 3340. * CV.LB_TO_KG + STD_CARGO_KG 
```